webpack/InsightsCloudSync/index.js

```diff
@@ -10,8 +10,5 @@ const mapStateToProps = state => ({});
 // map action dispatchers to props
 const mapDispatchToProps = dispatch => bindActionCreators(actions, dispatch);
 
-// export reducers
-export const reducers = {};
```

Comment:
I guess we can actually keep this, currently it doesn't affect anything,
but in case we will want to add a reducer to it, we can just fill this object.

Reply:
Added the reducer to the export